src/main/java/net/dries007/tfc/world/classic/worldgen/WorldGenOreVeins.java

```diff
@@ -58,6 +58,11 @@
     {
         Random rand = new Random(worldSeed + chunkX * 341873128712L + chunkZ * 132897987541L);
         List<Vein> veins = new ArrayList<>();
+        
```

Comment:
I said this before - this is a dirty hack that doesn't actually solve the issue here. Remove.